sys-libs/llvm-libunwind/llvm-libunwind-3.8.1.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+DESCRIPTION="C++ runtime stack unwinder from LLVM"
+HOMEPAGE="https://github.com/llvm-mirror/libunwind"
+if [ "${PV%9999}" = "${PV}" ] ; then
+	KEYWORDS="~amd64"
+	SRC_URI="http://llvm.org/releases/${PV}/libunwind-${PV}.src.tar.xz"
+else
+	KEYWORDS=""
+	SRC_URI=""
+fi
+
+LICENSE="|| ( UoI-NCSA MIT )"
+SLOT="0"
+IUSE="+static-libs"
+
+DEPEND=""
+RDEPEND="!sys-libs/libunwind"
+
+src_unpack() {
+	default
+	S="${WORKDIR}/libunwind-${PV}.src"
```

Comment:
The standard practice is to set it in global scope. It'd also make sense to define MY_P and use it to form both SRC_URI and S.